Sources/Amplitude/Amplitude.m

```diff
@@ -833,6 +901,36 @@ - (void)logRevenueV2:(AMPRevenue *)revenue {
     [self logEvent:kAMPRevenueEvent withEventProperties:[revenue toNSDictionary]];
 }
 
+#pragma mark - Deep link methods
+- (void)continueUserActivity:(NSUserActivity *)activity {
+    if (!self.defaultTracking.deepLinks) {
+        return;
+    }
+
+    if ([activity.activityType isEqualToString:NSUserActivityTypeBrowsingWeb]) {
+        NSString *urlString = activity.webpageURL.absoluteString;
+        NSString *referrerString = nil;
+        if (@available(iOS 11, tvOS 11.0, macOS 10.13, watchOS 4.0, *)) {
```

Comment:
Question for other versions, we will leave referrerString to nil?

Reply:
Yep, for other version, I didn't find a way to get the `referrerString`, so leave it as nil here.

Reply:
Should we remove the event properties in this case in stead of leave it as nil?

Reply:
Good question, when I tested it, I think when sending the event, the nil event property is filtered out already, see this [example event](https://app.amplitude.com/analytics/marvin-test/project/457920/search/amplitude_id%3D666031858690?eventFilter=%5BAmplitude%5D%20Deep%20Link%20Opened&collapsed=true&sessionHandle=YkAXHzQ_YkAXHzQ_JsSl6QC_BvzA_4515C7C9-6D85-4803-9AA8-D5A50A63FEC0&eventId=dd172adb-463b-4ae6-88ca-c156f2564ab8). I am not sure which part of our code did it actually (suspecting this [code](https://github.com/amplitude/Amplitude-iOS/blob/9bf9664199d9c5ea7367205180e9ee3c52a9aaa7/Sources/Amplitude/AMPUtils.m#L59)). Also, I think it is not an issue as the TS SDK applies similar strategy, see [code](https://github.com/amplitude/Amplitude-TypeScript/blob/83d987ce7ac384ec27d6534ad6af7a09a9a2058c/packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts#L24-L35).